doc/reference/configuration/configuration_reference.rst

```diff
@@ -3599,6 +3599,290 @@ The ``replication`` section defines configuration parameters related to :ref:`re
     | Default: ``false``
     | Environment variable: TT_REPLICATION_ANON
 
+.. _configuration_reference_replication_autoexpel:
+
+.. confval:: replication.autoexpel
+
+    **Since:** :doc:`3.3.0 </release/3.3.0>`
+
+    The ``replication.autoexpel`` option designed for managing dynamic clusters using YAML-based configurations.
+    It enables the automatic expulsion of instances that are removed from the YAML configuration.
+
+    Only instances with names that match the specified prefix are considered for expulsion; all others are excluded.
+    Additionally, instances without a persistent name are ignored.
+
+    If an instance is in read-write mode and has the latest database schema, it initiates the expulsion of instances that:
+
+	- Match the specified prefix
+	- Absent from the updated YAML configuration
+
+    The expulsion process follows the standard procedure, involving the removal of the instance from the ``_cluster`` system space.
+
+    The ``autoexpel`` logic is activated during specific events:
+
+	- **Startup**. When the cluster starts, ``autoexpel`` checks and removes instances not matching the updated configuration.
+	- **Reconfiguration**. When the YAML configuration is reloaded, ``autoexpel`` compares the current state to the updated configuration and performs necessary expulsions.
+	- ``box.status`` **watcher event**. Changes detected by the ``box.status`` watcher also trigger the ``autoexpel`` mechanism.
+
+
+    ``autoexpel`` does not take any actions on newly joined instances unless one of the triggering events occurs.
+    This means that an instance meeting the ``autoexpel`` criterion can still join the cluster, but it may be removed
+    later during reconfiguration or on subsequent triggering events.
+
+    ..  NOTE::
+        The ``replication.autoexpel`` option governs the expelling process and is configurable at the replicaset, group, and
+        global levels. It is not applicable at the instance level.
+
+
+    Configuration fields
```

Comment:
Similarly to https://github.com/tarantool/doc/pull/4742#discussion_r1928718995: it seems unusual to have a title-alike text without any formatting that would differentiate it from a usual text. If there is no appropriate header level in the markup, I would at least mark it semi-bold.

Reply:
fixed